src/main/scala/com/ironcorelabs/EncryptedPrivateKey.scala

```diff
@@ -0,0 +1,11 @@
+package com.ironcorelabs.scala.sdk
+
+import scodec.bits.ByteVector
+import com.ironcorelabs.{sdk => jsdk}
+
+case class EncryptedPrivateKey(bytes: ByteVector)
+
+object EncryptedPrivateKey {
+  def apply(bytes: Array[Byte]): EncryptedPrivateKey = EncryptedPrivateKey(ByteVector.view(bytes))
```

Comment:
Is this needed? There's nothing you can do with an `EncryptedPrivateKey` that you construct from bytes, is there?

Reply:
Sure. The real constructor was calling this one, but I just compressed them into one now.